### ISSUE TITLE
Spotify stopped working

### ISSUE THREAD

**Issue by brianegge** [owntone/owntone-server#1456]
Spotify was working fine for months. Then it stopped.

Log didn't show any errors, but each track failed to play.
```
[2022-04-24 07:44:11] [DEBUG]      mpd: Asynchronous listener callback called with event type 2.
[2022-04-24 07:44:11] [DEBUG]   player: Opening track: 'spotify:track:618wOV2U1XbUrhJqw4PwIW' (id=23638, seek=0)
[2022-04-24 07:44:11] [DEBUG]      web: notify callback reason: 71
[2022-04-24 07:44:11] [DEBUG]      mpd: Notify clients waiting for idle results: 2
[2022-04-24 07:44:11] [ WARN]   player: now 0, item_id 23638, now item_id 0
[2022-04-24 07:44:11] [DEBUG]       db: Running query 'BEGIN TRANSACTION;'
[2022-04-24 07:44:11] [DEBUG]      web: notify callback reason: 11
[2022-04-24 07:44:11] [DEBUG]      web: notify callback reply: 2
[2022-04-24 07:44:11] [DEBUG]       db: Starting enum 'SELECT * FROM queue f WHERE id = 23638 ORDER BY pos;'
[2022-04-24 07:44:11] [DEBUG]      web: notify callback reason: 11
[2022-04-24 07:44:11] [DEBUG]      web: notify callback reply: 2
```

Going to spotify.com yielded:
```
spotify upstream connect error or disconnect/reset before headers. reset reason: connection failure
```

I assume my IP got blacklisted. *I logged out of spotify on my laptop on the same network, logged back in, and it resolved the issue. 

Posting here in case someone else runs into the same.

**Comment by brianegge** [comment]
**I logged out of spotify on my laptop on the same network, logged back in, and it resolved the issue.**

**Comment by ejurgensen** [comment]
Strange that the log doesn't show anything. I also ran into Spotify not working any more, but in my case the logs did show an error. The cause was that a Spotify access point had gone into a bad state, and it stayed that way for weeks. So I added [a change](https://github.com/owntone/owntone-server/commit/48051f6364af1dcede260223d50629eba22975c7) so that OwnTone would select another access point if the first one fails.
